com.github.sdbg.debug.ui/src/com/github/sdbg/debug/ui/internal/chrome/ChromeLaunchShortcut.java

```diff
@@ -69,6 +69,10 @@ protected ILaunchConfigurationType getConfigurationType() {
 
   @Override
   protected void launch(IResource resource, String mode) {
+    launch(resource, mode, null);
+  }
+
+  public void launch(IResource resource, String mode, String url) {
```

Comment:
I wasn't after the shortcut but the logic in the launch process, which probably is better being in a utility. 

In the gwt plugin I was aiming for adding a menu to launch...
![61682cde-f18f-11e4-9cae-1e1265886a90](https://cloud.githubusercontent.com/assets/1326504/7477151/d0531b04-f306-11e4-8a00-6e484601fc13.png)